### PR TITLE
handle switch with no priority group and queue

### PR DESCRIPTION
**Pull request by lguohan**

### CODE REVIEW

File: orchagent/portsorch.cpp [hunk 1]
```diff
@@ -600,6 +600,11 @@ void PortsOrch::initializeQueues(Port &port)
     }
     SWSS_LOG_INFO("Get %d queues for port %s", attr.value.u32, port.m_alias.c_str());
 
+    if (attr.value.u32 == 0)
+    {
+        return;
+    }
+
     port.m_queue_ids.resize(attr.value.u32);
```
Comment (qiluo-msft) [inline orchagent/portsorch.cpp]: Move .resize above if-block to save some memory?

File: orchagent/portsorch.cpp [hunk 2]
```diff
@@ -629,6 +634,11 @@ void PortsOrch::initializePriorityGroups(Port &port)
     }
     SWSS_LOG_INFO("Get %d priority groups for port %s", attr.value.u32, port.m_alias.c_str());
 
+    if (attr.value.u32 == 0)
+    {
+        return;
+    }
+
     port.m_priority_group_ids.resize(attr.value.u32);
```
Comment (qiluo-msft) [inline orchagent/portsorch.cpp]: The same.

Comment (lguohan) [inline orchagent/portsorch.cpp]: updated

File: orchagent/portsorch.cpp (hunk 1 above)
Comment (lguohan) [inline orchagent/portsorch.cpp]: updated

**Pull request merged**